pkgs/development/libraries/libindicator/default.nix

```diff
@@ -0,0 +1,48 @@
+{ stdenv, fetchurl, lib, file
+, pkgconfig
+, gtkVersion, gtk2 ? null, gtk3 ? null }:
+
+with lib;
+
+stdenv.mkDerivation rec {
+  name = "libindicator-gtk${gtkVersion}-${version}";
+  version = "${versionMajor}.${versionMinor}";
+  versionMajor = "12.10";
+  versionMinor = "1";
+
+  src = fetchurl {
+    url = "${meta.homepage}/${versionMajor}/${version}/+download/libindicator-${version}.tar.gz";
+    sha256 = "b2d2e44c10313d5c9cd60db455d520f80b36dc39562df079a3f29495e8f9447f";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+
+  buildInputs = [ (if gtkVersion == "2" then gtk2 else gtk3) ];
+
+  postPatch = ''
+    substituteInPlace configure \
+      --replace 'LIBINDICATOR_LIBS+="''$LIBM"' 'LIBINDICATOR_LIBS+=" ''$LIBM"'
```

Comment:
You are right, it is unnecessary here. Thanks for pointing it out to me! I have updated the commit to remove the unnecessary escaping of the dollar signs.